### PR TITLE
Add block woo loyal customer block and Add related product block

### PR DESCRIPTION
**Pull request by RobStino**
Add WooCommerce Loyal Customer block files.

### CODE REVIEW

File: blocks/woo-loyal-customer/block.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
+.gcb-loyal-customer {
+    position: relative;
+    border-radius: 20px;
+    text-align: center;
+    overflow: hidden;
+    background-position: center;
+}
+
+.gcb-loyal-customer h2 {
+    font-size:  44px;
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.css]: Remove extra space

File: blocks/woo-loyal-customer/block.css [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    background-color: #dae0df;
+}
+
+.gcb-loyal-customer button {
+    position: relative;
+    display: flex;
+    align-items: center;
+    height: 50px;
+    border-radius: 3px;
+    background-color:#ea533c;
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.css]: Add space between `property: value`

File: blocks/woo-loyal-customer/block.css [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    color:#ffffff;
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.css]: Add space between `property: value`

File: blocks/woo-loyal-customer/block.php [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?php
+// Minimum number of previous purchases to see the coupon
+$min_previous_purchases = block_value('previous-purchases');
+
+// Current Customer Previous Purchases
+$customer_orders = get_posts([
+    'numberposts' => 10, // one order is enough
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.php]: - [ ] Align indentation
- [ ] Start inline comments with a Capital
- [x] End inline comment with a `.`
```
    'numberposts' => 10, // One order is enough.
    'meta_key'       => '_customer_user',
```

File: blocks/woo-loyal-customer/block.php [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<?php
+// Minimum number of previous purchases to see the coupon
+$min_previous_purchases = block_value('previous-purchases');
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.php]: Put spaces on both sides of the opening and closing parentheses (here and elsewhere in this file)

`block_value( 'previous-purchases' )`

File: blocks/woo-loyal-customer/block.php [hunk 6]
```diff
@@ -0,0 +1,37 @@
+<?php
+// Minimum number of previous purchases to see the coupon
```
Comment (cdils) [inline blocks/woo-loyal-customer/block.php]: End comment with `.` (here and elsewhere in this file)

File: blocks/woo-related-products/block.php [hunk 7]
```diff
@@ -0,0 +1,65 @@
+<?php
+// Variables
+$number_of_products = block_value( 'number-of-products' );
+$selected_product = block_value( 'product' );
+$selected_product_id = $selected_product->ID;
+$selected_product_categories = get_the_terms( $selected_product_id, 'product_cat' );
+$selected_product_first_category_slug = $selected_product_categories[0]->slug;
+$grid_cols = 3;
+$product_tile_class = '';
```
Comment (cdils) [inline blocks/woo-related-products/block.php:9]: Align indentation

```
$xyz123 = 'thing';
$x      = 'something';

```

File: blocks/woo-related-products/block.php [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+if ( 1 === $number_of_products ) {
+    $product_tile_class = ' gcb-related-products__single-product';
+};
+
+// The Query
+$the_query = new WP_Query( 
+    array( 
+        'post_type' => 'product',
+        'posts_per_page' => $number_of_products,
+        'tax_query' => array(
```
Comment (cdils) [inline blocks/woo-related-products/block.php]: Align indentation

File: blocks/woo-related-products/block.php [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+                'taxonomy' => 'product_cat',
+                'field'    => 'slug',
+                'terms'    => $selected_product_first_category_slug,
+            ),
+        ),
+    ) 
+);
+
+// The Loop
+if ($the_query->have_posts()) {
```
Comment (cdils) [inline blocks/woo-related-products/block.php]: Add space before and after parenthesis

File: blocks/woo-related-products/block.php [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+        ?>
+
+        <div class="gcb-related-products__tile<?php echo esc_attr( $product_tile_class ); ?>">
+            <img src="<?php echo esc_attr( get_the_post_thumbnail_url() ); ?>" alt="">
+            <div>
+                <h4><?php the_title(); ?></h4>
+                <p>
+                <?php echo get_woocommerce_currency_symbol();
+                 $product = wc_get_product(get_the_ID());
+                 echo $product->get_price(); //will give raw price
```
Comment (cdils) [inline blocks/woo-related-products/block.php]: Comment formatting.

`// Will give raw price.`

**Review by cdils — CHANGES_REQUESTED** [review]
Overall looks great! Just a few nitpicky formatting items to align with [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/.)

**Comment by RobStino** [comment]
Great review @cdils ! Thanks for taking the time to make those fixes!

**Review by RobStino — COMMENTED** [review]
These all look good!
Thanks again.
@cdils Can you approve the PR when you're able.

**Comment by cdils** [comment]
No problem! Original review was just noting needed changes, but I went ahead and applied them. 

I'd recommend using [WP Coding Standards](https://github.com/WordPress/WordPress-Coding-Standards) moving forward since this is code we'll be distributing to the community.

**Pull request merged**